src/lib/directives/mobile.directive.ts

```diff
@@ -0,0 +1,79 @@
+import { BreakpointObserver } from '@angular/cdk/layout';
+import { Directive, TemplateRef, ViewContainerRef, OnDestroy, OnInit, Input, DebugElement, EmbeddedViewRef } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+/**
+ * Directiva Mobile para evitar el render de elementos en dispositivos mobiles o viceversa.
+ * Funciona como un *ngIf, el elemento no se crea directamente.
+ *
+ * @example
+ * <plex-button *mobile></plex-button>
+ *
+ */
+@Directive({
+    // tslint:disable-next-line:directive-selector
+    selector: '[mobile]'
+})
+export class MobileDirective<T> implements OnDestroy, OnInit {
+    private openSubscription: Subscription;
+    private viewRef: EmbeddedViewRef<any> = null;
+
+    /**
+     * Determina si se muestra el elemento en mobile o no.
+     * @default true
+     */
+    @Input() mobile = true;
+
+    constructor(
+        private view: ViewContainerRef,
+        private nextRef: TemplateRef<ObserveContext<T>>,
+        private breakpointObserver: BreakpointObserver
+    ) { }
+
+
+    ngOnDestroy() {
+        this.openSubscription.unsubscribe();
+    }
+
+    ngOnInit() {
+        if (this.mobile === null) {
+            this.mobile = true;
+        }
+        const MOBILE_BREKPOINT = '(max-width: 599px)';
```

Comment:
Jaja es una boludez ortográfica, pero me dio a pensar si podríamos tener algún arreglo de constantes con los breakpoints? 
Porque podríamos tener cosas mobile only, tablet only, desktop only... de hecho la directiva podría ser "devices" en lugar de "mobile" :thinking: abro la discusión
```suggestion
        const MOBILE_BREAKPOINT = '(max-width: 599px)';
```

Reply:
Si lo pensé, pero me gustan los shorcuts, son más fáciles de recordar. En algún caso muy rebuscado saldrá la directiva general **ANY devices**.

Reply:
Bien, buenísimo entonces! tampoco hay que perder de vista que "devices" son pantallas grandes (turneros, etc).